src/answer/answer.controller.ts

```diff
@@ -82,6 +82,13 @@ export class AnswerController {
     @Param('id', ParseIntPipe) id: number,
   ): Promise<CreateAnswerRespondDto> {
     const userId = this.authService.verify(auth).userId;
+    this.authService.audit(
+      auth,
+      AuthorizedAction.create,
+      userId,
+      'answer',
```

Comment:
Parameter 'ownerId' should NOT be get from token!
This code leads to security vulnerability!